src/cpp/lpnamer/main/ProblemGeneration.cpp

```diff
@@ -187,6 +223,8 @@ void ProblemGeneration::RunProblemGeneration(
       antares_version < first_version_without_variables_files;
   (*logger)(LogUtils::LOGLEVEL::INFO)
       << "rename problems: " << std::boolalpha << rename_problems << std::endl;
+
+
   auto files_mapper = FilesMapper(antares_archive_path, xpansion_output_dir);
   auto mpsList = files_mapper.MpsAndVariablesFilesVect();
```

Comment:
Should we do this in ANTARES_API mode ?

Reply:
Renaming? Probably not. Adding a todo in #753

Reply:
No I meant creating the `mpsList` object (it seems unused in ANTARES_API mode) ?